Makefile-ocp.mk

```diff
@@ -0,0 +1,12 @@
+include Makefile
+
+.PHONY: verify
+verify: manifests
```

Comment:
```suggestion
verify-ocp: manifests
```

so it does not get into conflict with the Makefile verify

Reply:
This one is tricky. It first runs the `verify` in Makefile and after that runs the `verify` in here. No conflict (I learn this while working on this :))

Reply:
I checked it now. The reality is reverse. First verify in makefile-ocp and after the verify in makefile.

Reply:
> I see. Sgtm, even though such separation might be useful later.

This will be effective now without any change in openshift/release. I can rename it to `verify-ocp` when I add e2e tests. Because at that time I'll have to touch o/release anyway.

Reply:
Ok. Technically we do not control what upstream has in their makefile, so the separation would make sense.

We could also add it to both targets verify and verify-ocp now. And then change it in o/release

Reply:
Make sense. Updating now.

Reply:
Opened this https://github.com/openshift/release/pull/62941